tests/test_containers.py

```diff
@@ -63,7 +63,7 @@ def test_singularity_basic(fileutils):
 
     script = fileutils.get_test_conf_path("sleep.py")
     model.attach_generator_files(to_copy=[script])
-    exp.generate(model)
+    exp.generate(model, overwrite=True)
```

Comment:
Why is `overwrite=True` needed? Theoretically, test directories are wiped out before test execution (and after, if all tests passed). The experiment tree should not be there.

Reply:
While running the tests manually (i.e. `pytest test_containers.py`), the output didn't seem to be consistently cleared. Happy to take this out if this was some weird edge case and/or detrimental to debugging test failures.

Reply:
Yep, you were totally right...I must have just phase shifted into a parallel universe briefly.